netmanager/src/config/constants.js

```diff
@@ -260,16 +151,28 @@ const defaultConfig = {
 
 function runConfig(env) {
   console.log(`running in ${env} mode`);
-  switch (env) {
-    case "development":
-      return devConfig;
-    case "stage":
-      return stageConfig;
-    case "production":
-      return prodConfig;
-    default:
-      return prodConfig;
-  }
+  const configMapper = {
+    production: createAbsoluteUrls(
+      process.env.PROD_BASE_URL || "http://34.78.78.202/",
```

Comment:
is it okay to treat `http://34.78.78.202/` as a constant and just call it here? -  I think it will make it cooler :)

Reply:
ok, I think this fine - no need to make changes